helix-term/tests/test/commands.rs

```diff
@@ -722,5 +722,19 @@ fn foo() {
     ))
     .await?;
 
+    test((
+        indoc! {"\
+            #[a|]#
+            b
+            c
+            d
+            e
+            f
+            "},
+        "4xs\\n<ret>r,",
```

Comment:
I think it would be clearer here to see what we're intending to test here if we start with the selection on the newlines rather than using `4xs\\n<ret>` to form a selection. I believe you can format it like:

```
a#[
|]#b#[
|]#c#[
```

etc